filament/backend/src/metal/MetalHandles.mm

```diff
@@ -1086,10 +1087,12 @@ BufferDescriptor b(nullptr, 0u, [](void* buffer, size_t size, void* user) {
         descriptor.stencilAttachment.slice = 0;
         const bool discard = any(discardFlags & TargetBufferFlags::STENCIL);
         if (!discard) {
+            assert_invariant(context->supportsAutoDepthResolve);
             descriptor.stencilAttachment.resolveTexture = stencilAttachment.getTexture();
             descriptor.stencilAttachment.resolveLevel = stencilAttachment.level;
             descriptor.stencilAttachment.resolveSlice = stencilAttachment.layer;
             descriptor.stencilAttachment.storeAction = MTLStoreActionMultisampleResolve;
+            descriptor.stencilAttachment.stencilResolveFilter = MTLMultisampleStencilResolveFilterSample0;
```

Comment:
I think this will generate a compiler warning since `stencilResolveFilter` is only available on iOS 12+. (and we support iOS 11+). We may need to surround with `@available`.

Reply:
That's right, fixed it in https://github.com/google/filament/pull/6220/commits/dd6f2f2dbfb25c4456c6a4e9b739ffac24a50fc4.